<launch>

   <arg name="use_sim_time" default="true"/>
   <!--<param name="robot_description" textfile="$(find plastun_description)/urdf/plastun.urdf" /> -->
  <param name="robot_description"
	 command="$(find xacro)/xacro.py '$(find strirus_robot_description)/urdf/strirus.xacro'" /> 

<node name="generating_terrain" pkg="strirus_gazebo" type="generating_terrain.py" output="screen">
		<param name="cage_height_range_begin" value="0.1"/>
		<param name="cage_height_range_end" value="0.6" />
		<param name="cage_width_and_lengh" value="1.0"/> <!-- need to be float type -->
		<param name="cell_width_number" value="8"/>
		<param name="cell_length_number" value="15" />
		<param name="cage_height_param" value="rand"/>
		<param name="file_path" value="$(find strirus_gazebo)/maps/Generated_terrain/model.sdf"/>

	</node>

<!-- запускаем комнату -->
  <include file="$(find gazebo_ros)/launch/empty_world.launch">
    <arg name="world_name" value="$(find strirus_gazebo)/worlds/testing_area.world"/>
    <arg name="paused" value="true"/>
    <arg name="physics" default="ode"/>
    <arg name="verbose" default="true"/>
    <arg name="use_sim_time" value="$(arg use_sim_time)"/>
    
  </include>
  

<!-- Spawn a robot into Gazebo -->
<!--<node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" args="-file $(find plastun_description)/urdf/plastun.urdf -urdf -z 0.001 -model plastun" /> -->
  <node name="spawn_urdf" pkg="gazebo_ros" type="spawn_model" respawn="false" output="screen"
	args="-urdf -model strirus -param robot_description"/>

<!-- ros_control rorbot launch file -->
 <!-- <include file="$(find plastun_control)/launch/plastun_control.launch"/> -->


</launch>
